fractions/jaeger/src/main/resources/modules/io/jaegertracing/main/module.xml

```diff
@@ -1,5 +1,6 @@
 <module xmlns="urn:jboss:module:1.3" name="io.jaegertracing">
   <resources>
+    <artifact name="io.jaegertracing:jaeger-client:${version.jaeger}"/>
```

Comment:
Do you need this? I think this artifact is empty, apart from a dummy Version class